BitmarkRegistry/Controllers/Main/Properties/PropertiesViewController.swift

```diff
@@ -233,7 +231,10 @@ extension PropertiesViewController {
     let welcomeLabel = CommonUI.pageTitleLabel(text: "WELCOME TO BITMARK!")
     welcomeLabel.textColor = .mainBlueColor
 
-    let descriptionLabel = CommonUI.descriptionLabel(text: "Register, track, and trade property rights for your digital assets.")
+    let descriptionLabel = CommonUI.descriptionLabel(text:
```

Comment:
We can use `Multiline String Literals` for this. See: https://docs.swift.org/swift-book/LanguageGuide/StringsAndCharacters.html